frontend/guardian.rb

```diff
@@ -98,30 +99,14 @@ def rejected_beep(gpio_command)
     tag = $2
   end
 
-  tag_log = {
+  tag_log = TagLog.create(
     card_type: tag =~ /^c/ ? 'clipper' : 'rfid',
     card_number: tag,
     held_tag: flag == 'h' ? true : false
-  }
-
-  authorization = DoorAuthorization.first(card_number: tag)
-
-  if authorization
-    tag_log[:name] = authorization.name
-
-    unless authorization.expired? || ! authorization.active?
-      tag_log[:door_opened] = true
-    end
-  end
-
-  TagLog.create(tag_log)
+  )
 
-  if tag_log[:door_opened]
-    # open the door
-    open_door(gpio_command)
-    accepted_beep(gpio_command)
-    close_door(gpio_command)
-  else
-    rejected_beep(gpio_command)
+  if tag_log.door_opened
+    open_door(gpio_command, tag_log.door)
```

Comment:
Adding that back in after the door opens. Is it necessary/useful to have the .2s sleep that used to live in rejected_beep?